beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -407,6 +413,14 @@ func setExecution(blk interfaces.SignedBeaconBlock, execution interfaces.Executi
 		return errors.Wrap(err, errMessage)
 	}
 
+	// If the block version is below Electra, no further actions are needed
+	if blk.Version() < version.Electra {
+		return nil
+	}
+
+	if err := blk.SetExecutionRequests(requests); err != nil {
+		return errors.Wrap(err, errMessage)
```

Comment:
You have to update `errMessage` because it currently describes KZG commitments. But honestly I feel that it only makes sense to use `errMessage` for `SetExecution` because it removes duplication. It doesn't help to update it later only to use it in one place (in fact, it can lead to bugs such as this one).

Reply:
just adding my own custom error here